### DISCUSSION TITLE
Should I leave it 'true' or 'false' in Mirage.Enemies.cfg?

### DISCUSSION THREAD

**Discussion by Ghustavitto** [qwbarch/mirage#123] [Q&A] [answered]
If I leave it 'true' is it because the mod will work on this monster?
![image](https://github.com/user-attachments/assets/379a0049-4d64-4815-a83b-1a14edd3c061)

**Comment by qwbarch** [comment, marked as answer]
Good question, sorry for not being more specific in the config. Setting a monster to ``true`` means that monster will mimic players' voices, whereas setting it to ``false`` doesn't.

By default this means only ``Masked`` will mimic voices since it's the only monster set to ``true``.
